sh/install-dependencies.sh

```diff
@@ -3,10 +3,13 @@
 # PROGRAM BY PAPIT SASU, INSTALL THE REQUIRED DEPENDENCIES
 ################################################################################
 # 1) INSTALL DISTANT SYNAPTIC PACKAGES
-apt-get -y install libxml2-utils mysql-server tree libmysqlclient-dev python3-pip
+apt-get -y install libxml2-utils mariadb-server tree libmariadbclient-dev python3-pip
```

Comment:
Is it really safe to change the DB at this stage without the Pythoh client?
What is the benefit of MariaDB over MySQL?

Reply:
MariaDB is the new name for MySQL in Debian. It's a fork of MySQL when Oracle bought Sun who owned MySQL : https://www.ionos.fr/digitalguide/hebergement/aspects-techniques/mariadb-vs-mysql/